src/xrGame/saved_game_wrapper.h

```diff
@@ -22,6 +22,7 @@ class CSavedGameWrapper
     _LEVEL_ID m_level_id;
     shared_str m_level_name;
     float m_actor_health;
+    bool m_valid_save;
```

Comment:
Do we really need this boolean value? Looks like we can easily determine if the save is valid or not by inspecting the values of members m_level_id, m_level_name. So maybe it will be better to create and export a method like 'can_be_loaded' for determining validity instead of creating extra flags?